README.md

```diff
@@ -1,3 +1,101 @@
-# large-events
+# Large Events
 
 [![Build Status](https://travis-ci.com/knative-portability/large-events.svg?branch=master)](https://travis-ci.com/knative-portability/large-events)
+
+## Introduction
+
+An app built as proof of portability for [Knative](https://knative.dev) that provides a platform for event organizers to share info during large events like concerts, parades, expos, etc. Key features of this app include:
+* Allow authorized event organizers to create and edit events
+* Handle uploading and serving multimedia posts made by attendees
+* Organize and filter multiple sub-events
+* Provide a clean front-end web UI for interaction with the service
+* Authenticate with OAuth 2.0
+
+This app is built as part of a proof of portability project for [Knative](https://knative.dev). It is meant to show the key features of Knative, to test the conformance various cloud product implementations of Knative like [Google Cloud Run](https://cloud.google.com/run/), and to document with functioning sample code how one might develop, build, and deploy with Knative. Key features of Knative this app demonstrates include:
```

Comment:
I'd remove the Cloud Run reference here, and instead have a section on "where to run this?" that references that it's been tested on Cloud Run.

Reply:
(And then we can add to the list as we test it other places.)

Reply:
The "Deployment" section seems to cover this nicely.

Reply:
Done https://github.com/knative-portability/large-events/pull/6/commits/5a3aea8b819ebec42eb9f5e5dfa6b57c68588a4c